src/types/inventory.ts

```diff
@@ -345,6 +342,7 @@ export interface ProductDashboardInterface {
     product_status: {
       status_id: number | string;
       status_name: string;
+      status_slug:string;
```

Comment:
@jerlyrosa format , give it the same space as the rest

Reply:
done